src/fides/api/ctl/database/session.py

```diff
@@ -15,7 +16,7 @@
 sync_engine = create_engine(CONFIG.database.sync_database_uri, echo=False)
 sync_session = sessionmaker(
     sync_engine,
-    class_=Session,
+    class_=ExtendedSession,
```

Comment:
This in will run `session.rollback()` automatically on every failed transaction.